packages/platforms/browser/lib/default-routing-provider.ts

```diff
@@ -2,7 +2,7 @@ import { type OnSettle } from './on-settle'
 import getAbsoluteUrl from './request-tracker/url-helpers'
 import { type RouteResolver, type RoutingProvider, type StartRouteChangeCallback } from './routing-provider'
 
-const defaultRouteResolver: RouteResolver = (url: URL) => url.pathname
+const defaultRouteResolver: RouteResolver = (url: URL) => [null, undefined, ''].includes(url.pathname) ? url.toString() : url.pathname
```

Comment:
This is making a new array and iterating through it each time it's called

This would be more efficient and it's less code too:

```ts
(url: URL) => url.pathname || url.toString()
```